src/models/user_metadata.rs

```diff
@@ -40,9 +44,13 @@ pub struct UserMetadata {
     #[serde(rename = "last_active_at")]
     pub last_active_at: i64,
     #[serde(rename = "org_id_to_org_info", skip_serializing_if = "Option::is_none")]
-    pub org_id_to_org_info: Option<::std::collections::HashMap<String, crate::models::UserInOrg>>,
+    pub org_id_to_org_info: Option<HashMap<String, crate::models::UserInOrg>>,
     #[serde(rename = "legacy_user_id", skip_serializing_if = "Option::is_none")]
     pub legacy_user_id: Option<String>,
+    #[serde(rename = "impersonated_user_id", skip_serializing_if = "Option::is_none")]
+    pub impersonated_user_id: Option<String>,
```

Comment:
I don't think this field exists for `UserMetadata`, just for the `User` since that's in the access token.

Reply:
done.